assets/js/googlesitekit/widgets/util/combine-widgets.test.js

```diff
@@ -232,4 +232,13 @@ describe( 'combineWidgets', () => {
 			expected
 		);
 	} );
+
+	it( 'works with no widgets', () => {
+		const widgets = [];
+		const widgetStates = [];
+		const layout = getWidgetLayout( widgets, widgetStates );
+		expect( () =>
+			combineWidgets( widgets, widgetStates, layout )
+		).not.toThrow();
```

Comment:
You can see this test fail with the error in the issue before the early returns were added here https://github.com/google/site-kit-wp/issues/3618#issuecomment-892488288